homeassistant/components/ihc/service_functions.py

```diff
@@ -65,7 +60,7 @@ def _get_controller(call):
             if controller_conf[IHC_CONTROLLER_INDEX] == controller_index:
                 return controller_conf[IHC_CONTROLLER]
         # if not found the controller_index is ouf of range
-        _LOGGER.error("The controller index is of of range")
+        raise ValueError()
```

Comment:
Please include the message as exception argument.
```suggestion
        raise ValueError("The controller index is out of range")
```